nl/mail/chrome/messenger/cloudfile/management.dtd

```diff
@@ -1,8 +1,8 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this file,
    - You can obtain one at http://mozilla.org/MPL/2.0/.  -->
-<!ENTITY cloudfileMgmt.privacyPolicy "Privacy Policy">
-<!ENTITY cloudfileMgmt.termsOfService "Terms of Service">
-<!ENTITY cloudfileMgmt.usedSpace "Used Space:">
-<!ENTITY cloudfileMgmt.unusedSpace "Unused Space:">
-<!ENTITY cloudfileMgmt.upgradeOffer "Get more space">
+<!ENTITY cloudfileMgmt.privacyPolicy "Privacybeleid">
+<!ENTITY cloudfileMgmt.termsOfService "Servicevoorwaarden">
+<!ENTITY cloudfileMgmt.usedSpace "Gebruikte Ruimte:">
+<!ENTITY cloudfileMgmt.unusedSpace "Ongebruikte Ruimte:">
```

Comment:
CORRECTION NEEDED, depending on context. The translation is correct if we're looking at this word without any context. If we're talking about space in the online storage services context, usually ''Opslagruimte'' is used vs ''Ruimte'.
In that case, this line SHOULD BE: "Ongebruikte Opslagruimte:"